### PR TITLE
fix(flags): Update change tracking docs to highlight new settings page

### PR DESCRIPTION
**Pull request by cmanallen**
## DESCRIBE YOUR PR
*Tell us what you're changing and why. If your PR **resolves an issue**, please link it so it closes automatically.*

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [x] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [ ] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: docs/organization/integrations/feature-flag/launchdarkly/index.mdx [hunk 1]
```diff
@@ -20,14 +20,15 @@ Sentry can track changes to feature flag definitions and report suspicious featu
...
-Enabling Change Tracking is a two step process. To get started visit the [feature-flags settings page](https://sentry.io/orgredirect/organizations/:orgslug/settings/feature-flags/) in a new tab. Then follow the steps listed below.
+Enabling Change Tracking is a three step process. To get started visit the [feature-flags settings page](https://sentry.io/orgredirect/organizations/:orgslug/settings/feature-flags) in a new tab. Then follow the steps listed below.
 
-1. **Register the webhook URL**.
-    - Under the "Integrate Feature Flag Service" heading a Sentry webhook URL has been provided.
-    - Copy the Sentry webhook URL revealed in the fly out and paste it into LaunchDarkly within their [webhook integration UI](https://app.launchdarkly.com/settings/integrations/webhooks/new).
-2. **Set the Signing Secret**.
+1. **Click the "Add New Provider" button.
+    - One webhook secret can be registered per provider type.
+2. **Register the webhook URL**.
```
Comment (aliu39) [inline docs/organization/integrations/feature-flag/launchdarkly/index.mdx:27]: Where do we get this URL?

Comment (cmanallen) [reply docs/organization/integrations/feature-flag/launchdarkly/index.mdx:27]: Its on the page (or rather will be after michelle makes a change).

Comment (cmanallen) [reply docs/organization/integrations/feature-flag/launchdarkly/index.mdx:27]: On the "Add New Provider" form page.

Comment (aliu39) [reply docs/organization/integrations/feature-flag/launchdarkly/index.mdx:27]: Ok, could we have it on both form states? (Before/after submitting secret) cc @michellewzhang

**Review by aliu39 — APPROVED** [review]
Do we need this for OpenFeature too?

**Comment by cmanallen** [comment]
@aliu39 Nope!

**Comment by aliu39** [comment]
Oh yeah, no change tracking, sg!

**Pull request merged**